search/package.json

```diff
@@ -35,7 +35,7 @@
   },
   "scripts": {
     "debug": "export SLS_DEBUG=* && IS_LOCAL=true sls offline --config serverless.yml",
-    "start": "IS_LOCAL=true sls offline --config serverless.yml",
+    "start": "IS_LOCAL=true sls offline --config serverless.yml --protocol https",
```

Comment:
Does this require a certificate or does it have its own self-signed cert?

Reply:
Hmm, now that I look at this, the offline usage should be `http`, as that's all supported by the offline server.  Thanks, good catch, will fix.